Composer/packages/client/__tests__/components/notificationList.test.js

```diff
@@ -8,9 +8,9 @@ import { NotificationList } from '../../src/pages/notifications/NotificationList
 
 describe('<NotificationList/>', () => {
   const items = [
```

Comment:
why have id be a string but reference a number?

```javascript
{
  id: '1'
}
```

also, if any of these strings render to the user, they need to be wrapped in `formatMessage`